whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/DummifierRC.java

```diff
@@ -0,0 +1,75 @@
+package net.ripe.db.whois.common.rpsl;
+
+import com.google.common.collect.Lists;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class DummifierRC extends DummifierCurrent {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DummifierRC.class);
+
+    @Override
+    public RpslObject dummify(final int version, final RpslObject rpslObject) {
+        final List<RpslAttribute> attributes = Lists.newArrayList(rpslObject.getAttributes());
+
+        for (int i = 0; i < attributes.size(); i++) {
+            RpslAttribute replacement = attributes.get(i);
+
+            replacement = dummifyOrgName(rpslObject, replacement);
+            replacement = dummifyDescr(rpslObject, replacement);
```

Comment:
the `rpslObject` is also not used by the `dummifyDescr` method...consider removing/using it somehow?

Reply:
same observation applies to `dummifyRemarks`, `dummifyAsName` and `dummifyOwner` methods...

Reply:
I wanted a consistent method signature so I could extract those methods to sub-classes, but the class is very simple as-is, I removed the unused method arguments.